jumpscale/servers/threebot/threebot.py

```diff
@@ -468,6 +475,26 @@ def packages(self):
             self._packages = self._package_manager.get(self.instance_name)
         return self._packages
 
+    def check_dependencies(self):
+        install_msg = "Visit https://github.com/threefoldtech/js-sdk/blob/development/docs/wiki/quick_start.md for installation guide"
+
+        if not self.nginx.installed:
+            raise j.exceptions.NotFound(f"nginx is not installed.\n{install_msg}")
+        rc, _, _ = j.sals.process.execute("which certbot")
+        if rc:
+            raise j.exceptions.NotFound(f"certbot is not installed.\n{install_msg}")
+        rc, _, _ = j.sals.process.execute("dpkg -l python-certbot-nginx")
```

Comment:
if it's python you should be able to check by attempting an import or via resource_pkg instead

Reply:
It's an nginx plugin and installed using apt

Reply:
it's not an nginx plugin, it's infact a plugin for certbot to be able to manage nginx.
You can check if this plugin is installed by doing
try:
   import certbot_nginx
   its installed
except ImportError:
  no it's not installed